server/svix-server/src/v1/endpoints/application.rs

```diff
@@ -48,26 +48,31 @@ pub struct ApplicationIn {
     #[validate]
     #[serde(skip_serializing_if = "Option::is_none")]
     pub uid: Option<ApplicationUid>,
+
+    #[serde(default)]
+    pub metadata: Metadata,
 }
 
 // FIXME: This can and should be a derive macro
 impl ModelIn for ApplicationIn {
-    type ActiveModel = application::ActiveModel;
+    type ActiveModel = (application::ActiveModel, applicationmetadata::ActiveModel);
```

Comment:
I think that why biggest grip with this whole change is that it tries to take unrelated items that we update and fetch (app and metadata), and force-treat them as if they are one model. They are very related, and tied together, they could have maybe been a property on the model, but they aren't, and trying to force it feels off and causes a lot of confusing code.

Reply:
I ended up keeping this as is. The models are closely related, at least as far as the API is concerned. I don't really agree that this is confusing (it's quite explicit), and makes it obvious that the two models need to be updated together.